examples/keras/document-denoiser/README.md

```diff
@@ -0,0 +1,51 @@
+# Clean Dirty Documents w/ Autoencoders
+
+This example model cleans text documents of anything that isn't text (aka noise): coffee stains, old wear artifacts, etc. You can inspect the notebook that has been used to train the model [here](trainer.ipynb).
+
+Here's a collage of input texts and predictions.
+
+![Imgur](https://i.imgur.com/M4Mjz2l.jpg)
+
+*Figure 1 - The dirty documents are on the left side and the cleaned ones are on the right*
+
+## Sample Prediction
+
+### Dependencies
+
+First install the dependencies for the client locally
+```bash
+pip install opencv-python click numpy requests
+```
+
+Make sure this is run on *Python 3.x*.
+
+### Prediction
+
+Once this model is deployed, get the API endpoint by running `cortex get document-denoiser`. Then export the endpoint by running
+```bash
+export ENDPOINT=your-api-endpoint
+export IMAGE_URL=https://i.imgur.com/JJLfFxB.png
+```
+
+Now let's take a sample image like this one.
+
+![Imgur](https://i.imgur.com/JJLfFxB.png)
+
+Then run the following piped commands
+```bash
+curl "${ENDPOINT}" -X POST -H "Content-Type: application/json" -d '{"url":"'${IMAGE_URL}'"}' |
```

Comment:
If we put the URL in a sample.json
```
{
  "url": "https://i.imgur.com/JJLfFxB.png"
}
```
users would be able to get the exact curl command to run from `cortex get document-denoiser`

Reply:
I can do this. The reason I kept it inline was to make it easy to try another example and to prevent the necessity of having another file (`sample.json`). I don't have a strong opinion here. What do you think?

Reply:
We can create the sample.json but leave the instructions as is (using the exported variables). We can remain consistent with the other examples while still making it easy for users to try different images using env variables.